test_scripts/AppIconsFolder/ATF_N_AppIconsAmountToRemove_larger_than_icons_amount.lua

```diff
@@ -0,0 +1,245 @@
+  file=io.open(name,"r")
+  if file ~= nil then
+    io.close(file)
+    if messages == true then
+      commonFunctions:userPrint(32, "File " .. tostring(name) .. " exists")
+    end
+    return true
+  else
+    if messages == true then
+      commonFunctions:userPrint(31, "File " .. tostring(name) .. " does not exist")
+    end
+    return false
+  end
+end
+
+-- Generate path to application folder
+local function pathToAppFolderFunction(appID)
+  commonSteps:CheckSDLPath()
+  local path = config.pathToSDL .. tostring("storage/") .. tostring(appID) .. "_" .. tostring(config.deviceMAC) .. "/"
+  return path
+end
+
+local function folderSize(PathToFolder)
+  local sizeFolderInBytes
+  local aHandle = assert( io.popen( "du -sh " ..  tostring(PathToFolder), 'r'))
+  local buff = aHandle:read( '*l' )
+  local sizeFolder, measurementUnits = buff:match("([^%a]+)(%a)")
+  if measurementUnits == "K" then
+    sizeFolder  =  string.gsub(sizeFolder, ",", ".")
+    sizeFolder = tonumber(sizeFolder)
+    sizeFolderInBytes = sizeFolder * 1024
+  elseif
+    measurementUnits == "M" then
+    sizeFolder  =  string.gsub(sizeFolder, ",", ".")
+    sizeFolder = tonumber(sizeFolder)
+    sizeFolderInBytes = sizeFolder * 1048576
+  end
+  return sizeFolderInBytes
+end
+
+local function makeAppIconsFolderFull(AppIconsFolder)
+  local sizeToFull
+  local sizeAppIconsFolderInBytes = folderSize(config.pathToSDL .. tostring(AppIconsFolder))
+  local appIconsFolderMaxSize = 1048576
+  local oneIconSize = 326360
+  sizeToFull = appIconsFolderMaxSize - sizeAppIconsFolderInBytes
+  local i =1
+  while sizeToFull > oneIconSize do
+    os.execute("sleep 10")
+    local copyFileToAppIconsFolder = assert( os.execute( "cp files/icon.png " .. tostring(config.pathToSDL) .. tostring(AppIconsFolder) .. "/icon" .. tostring(i) ..".png"))
+    i = i + 1
+    if copyFileToAppIconsFolder ~= true then
+      commonFunctions:userPrint(31, " Files are not copied to " .. tostring(AppIconsFolder))
+    end
+     sizeAppIconsFolderInBytes = folderSize(config.pathToSDL .. tostring(AppIconsFolder))
+     sizeToFull = appIconsFolderMaxSize - sizeAppIconsFolderInBytes
+      if i > 10 then
+        commonFunctions:userPrint(31, " Loop is breaking due to a lot of iterations ")
+        break
+    end
+  end
+end
+
+local function checkFunction()
+  local aHandle = assert( io.popen( "ls " .. config.pathToSDL .. "Icons/" , 'r'))
+  local listOfFilesInStorageFolder = aHandle:read( '*a' )
+  commonFunctions:userPrint(32, "Content of storage folder: " ..tostring("\n" ..listOfFilesInStorageFolder))
+  local iconsFolder = config.pathToSDL .. tostring("Icons/")
+  local applicationFileToCheck = iconsFolder .. RAIParameters.appID
+  local applicationFileExistsResult = checkFileExistence(applicationFileToCheck)
+  if applicationFileExistsResult ~= true then
+    commonFunctions:userPrint(31, tostring(RAIParameters.appID) .. " icon is absent")
+    status = false
+  end
+  for i=1, 3, 1 do
```

Comment:
@aarotar, Step in code: `for i=1, 3, 1 do` can be omitted, so the following will work in the same way: `for i=1, 3 do`